.github/workflows/report.yml

```diff
@@ -82,15 +82,48 @@ jobs:
         with:
           version: ${{ inputs.bowtie-version }}
 
-      - name: Generate implementations.json file
+      - name: Generate implementations.json (Private API) file
         run: |
           bowtie info $(bowtie filter-implementations | sed 's/^/-i /') --format json > implementations.json
 
+      - name: Generate public-implementations (Public API) file
+        run: |
+          bowtie filter-implementations | \
```

Comment:
This is going to essentially immediately bite us, as this code is essentially incomprehensible. But really we need to get `bowtie site` to be a way of generating all of what we need from Python rather than writing YAML shell code. So I guess for right this second it's ok.